airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py

```diff
@@ -374,7 +374,13 @@ def load_schema(name: str, stream_options: Mapping[str, Any]) -> Tuple[str, Opti
                 ):
                     if err:
                         self.logger.error(f"Loading error of the {stream_name} schema: {err}")
-                        continue
+                        # Without schema information, the source can't determine the type of stream to instantiate and there might be issues
```

Comment:
I would have liked to link the stream that is causing the issue there but I don't see how as described [here](https://airbytehq-team.slack.com/archives/C02TL38U5L7/p1712943722694429)

Reply:
Waiting for https://github.com/airbytehq/airbyte/pull/37108 to be released so that I can update this